tools/vuln_valid/vulnValidate.js

```diff
@@ -31,7 +31,12 @@ const coreModel = joi.object().keys({
   type: joi.string().optional(),
   cvss_score: joi.number().optional(),
   cvss: joi.string().optional(),
-  reported_by: joi.string().optional()
+  reported_by: joi.string().optional(),
+  affectedEnvironments: joi
+  .array()
+  .items(joi.string().valid("all", "osx", "win", "linux"))
```

Comment:
I still think we need to have a bigger list here. Including all the supported environments

Reply:
I extended the list to include smartOS, AIX, FreeBSD in [17e08b9](https://github.com/nodejs/security-wg/pull/912/commits/17e08b9b7aa7238487d811532aa885b5f342b618), based on [the Platform list](https://github.com/nodejs/node/blob/c2e4b1fa9ad0b744616c4e4c13a5017772a630c4/BUILDING.md#platform-list)

Reply:
I think that will be better to use os.platform possible values rather the platform list, I will do a little refactor

Reply:
Refactor made in [535ce83](https://github.com/nodejs/security-wg/pull/912/commits/535ce830904cf1367620372ae3642175a14569a5)